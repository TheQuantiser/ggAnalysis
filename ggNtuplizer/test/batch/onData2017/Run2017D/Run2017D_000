file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/36D399F6-3037-E811-9A5C-001EC94BF70E.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/201B887A-A037-E811-8409-549F3525C318.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/42416F6A-A437-E811-83C2-74867AF198F0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/D0FB90B4-9A37-E811-B9E7-001EC951439E.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/7EA339F1-CD37-E811-A190-14187740D279.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/A4656888-3437-E811-B365-1866DAEA7E64.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/74F091CA-9037-E811-8283-B083FED42B3A.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/129B335C-8837-E811-B01C-1866DAEB5C74.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/A6BFAFCC-B037-E811-8A14-801844DEF0E0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/8EB74FAD-9637-E811-94B9-801844E561F8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/FC62D37F-3B37-E811-A155-801844E561C0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/2EB8B832-A837-E811-84E6-90B11C0BC86F.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/4CB0530B-BA37-E811-85F9-14187741278B.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/30000/329C6BB7-AB37-E811-ABE7-782BCB53A0BF.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/8A8CE5AC-8037-E811-A768-801844DEEF1C.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/08C25895-4437-E811-B27E-001C23C0C7A9.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/70BA90F2-8A37-E811-8156-A4BADB22B463.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/AE4EEFA7-4A37-E811-B192-1866DAEB1FC8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/E08790DC-7237-E811-8E8B-782BCB50ACEF.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/54394B41-6537-E811-9770-B083FED3F4E2.root
