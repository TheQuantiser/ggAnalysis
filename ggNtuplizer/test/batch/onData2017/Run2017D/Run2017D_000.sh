#!/bin/bash

hostname

source /cvmfs/cms.cern.ch/cmsset_default.sh

cd /local/cms/user/wadud/aNTGCmet/CMSSW_9_4_17/src/ggAnalysis/ggNtuplizer/test/ 

eval `scramv1 runtime -sh`

cmsRun --jobreport=/local/cms/user/wadud/aNTGCmet/CMSSW_9_4_17/src/ggAnalysis/ggNtuplizer/test/batch/onData2017//Run2017D//jobReport/Run2017D_000.xml /local/cms/user/wadud/aNTGCmet/CMSSW_9_4_17/src/ggAnalysis/ggNtuplizer/test/run_condor_data2017_94X.py InputFileList=/local/cms/user/wadud/aNTGCmet/CMSSW_9_4_17/src/ggAnalysis/ggNtuplizer/test/batch/onData2017//Run2017D/Run2017D_000 outputFile=/local/cms/user/wadud/aNTGCmet/ntuples/METv4//Run2017D_000.root maxEvents=-1
