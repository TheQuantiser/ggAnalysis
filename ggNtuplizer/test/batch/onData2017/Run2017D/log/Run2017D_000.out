scorpion10.spa.umn.edu
